<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>781</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="stepButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Step</string>
    </property>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>540</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pauseButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>540</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QSlider" name="speedSlider">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>530</y>
      <width>421</width>
      <height>20</height>
     </rect>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="slowerLabel">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>530</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Slower</string>
    </property>
   </widget>
   <widget class="QLabel" name="fasterLabel">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>530</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Faster</string>
    </property>
   </widget>
   <widget class="QLabel" name="speedLabel">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>550</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Speed:</string>
    </property>
   </widget>
   <widget class="QLabel" name="turnLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Turn: </string>
    </property>
   </widget>
   <widget class="QLabel" name="popLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Population: </string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="histGraphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>781</width>
      <height>200</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>20</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="emptyButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>20</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Empty</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
